Packages/com.unity.inputsystem/InputSystem/Actions/InputActionState.cs

```diff
@@ -3697,7 +3697,7 @@ internal static void OnDeviceChange(InputDevice device, InputDeviceChange change
                     continue;
                 if (change == InputDeviceChange.UsageChanged && !state.IsUsingDevice(device) && !state.CanUseDevice(device))
                     continue;
-                if (change == InputDeviceChange.ConfigurationChanged && !state.IsUsingDevice(device))
+                if (change == InputDeviceChange.ConfigurationChanged && !state.IsUsingDevice(device) && !state.CanUseDevice(device))
```

Comment:
Please add comment why `CanUseDevice` is present on this branch. This it's sufficiently non-obvious to warrant a quick blurb.